Tools/autotest/default_params/copter-dodecahexa.parm

```diff
@@ -0,0 +1,94 @@
+EK2_ENABLE      1
+FRAME_TYPE	0
+MAG_ENABLE	1
+FS_THR_ENABLE   1
+BATT_MONITOR    4
+CH7_OPT         7
+COMPASS_LEARN   0
+COMPASS_OFS_X   5
+COMPASS_OFS_Y   13
+COMPASS_OFS_Z   -18
+COMPASS_OFS2_X   5
+COMPASS_OFS2_Y   13
+COMPASS_OFS2_Z   -18
+FENCE_RADIUS    150
+FRAME_CLASS     12
+RC1_MAX         2000.000000
+RC1_MIN         1000.000000
+RC1_TRIM        1500.000000
+RC2_MAX         2000.000000
+RC2_MIN         1000.000000
+RC2_TRIM        1500.000000
+RC3_MAX         2000.000000
+RC3_MIN         1000.000000
+RC3_TRIM        1500.000000
+RC4_MAX         2000.000000
+RC4_MIN         1000.000000
+RC4_TRIM        1500.000000
+RC5_MAX         2000.000000
+RC5_MIN         1000.000000
+RC5_TRIM        1500.000000
+RC6_MAX         2000.000000
+RC6_MIN         1000.000000
+RC6_TRIM        1500.000000
+RC7_MAX         2000.000000
+RC7_MIN         1000.000000
+RC7_TRIM        1500.000000
+RC8_MAX         2000.000000
+RC8_MIN         1000.000000
+RC8_TRIM        1500.000000
```

Comment:
actually with AC3.5 (and higher) we don't use the input ranges for the outputs anymore.  We now have SERVOx_ parameters.